app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/upload_mandatory_evidence.scala.html

```diff
@@ -105,7 +104,6 @@
           errorKey = None,
           labelAsHeading = false,
           labelClass = Some("heading-medium bold"),
-          autoComplete = AutoCompleteType.On,
```

Comment:
will this autocompletion work without this flag?

Reply:
autocomplete doesn't work on file inputs, so it is invalid on this element